app/controllers/api/arbitration_profiles_controller.rb

```diff
@@ -0,0 +1,45 @@
+module Api
+  class ArbitrationProfilesController < BaseController
+    def create_resource_arbitration_profiles(_type, _id, data)
```

Comment:
@abellotti in the future this would be nice if it was just `#create` but we're not quite there yet. Future PR!